content/management/configuration-api/v3.4/index.mdx

```diff
@@ -2438,6 +2438,7 @@ Registers a webhook for the Client ID (application) provided in the request. Lat
 | `filters.chat_presence.user_ids.values`         | no       | `[]string` | Array of Users' ids; if any specified User is in the chat, the webhook will be triggered.                                                                                                      |
 | `filters.chat_presence.user_ids.exclude_values` | no       | `[]string` | Array of Users' ids; if any specified Users is in the chat, the webhook will not be triggered.                                                                                                 |
 | `filters.chat_presence.my_bots`                 | no       | `bool`     | If any bot owned by `owner_client_id` is in the chat, the webhook will be triggered.                                                                                                           |
+| `filters.source_type`                           | no       | `[]string` | Filters to check if a webhook should be triggered based on webhook's source. Possible values in array : `my_client`, `other_clients`, `system`.                                                |
```

Comment:
IMO this note should be similar to other filters'

Reply:
_Array of source types. Possible values: `my_client`, `other_clients`, `system`; if webhook source match, the webhook will be triggered._
something more like this?

Reply:
How about
> Array of source types. Possible values: my_client, other_clients, system; if webhook's source client_id matches, the webhook will be triggered.

Reply:
Isn't it confusing that here we have source types, and here we are talking about client_id? Is it known that we're recognizing source of webhook by client_id?

Reply:
maybe we'll need some additional explaination as `webhook source` is not defined anywhere and it's hard to deduct that it's connected to client_id of the request that triggered the webhook

Reply:
hey! I suggest:
> Array of source types. Possible values: `my_client` (client ID of your app), other_clients (client IDs other than your app's), system (no client ID); if the source which triggered the webhook matches the filter, the webhook will be sent.

This way we would guide the user on what is a webhook source without a separate disclaimer. I was a little concerned that my_client may not indicate that we refer to the client_id thus the explanations in brackets